app/engine/engine.ts

```diff
@@ -47,19 +47,23 @@ export function getWinningBuzzer(buzzes: Map<string, number>):
       deltaMs: number;
     }
   | undefined {
-  const result = Array.from(buzzes.entries()).reduce(
-    (acc, [userId, deltaMs]) => {
-      if (
-        deltaMs !== CANT_BUZZ_FLAG &&
-        deltaMs < acc.deltaMs &&
-        deltaMs <= CLUE_TIMEOUT_MS
-      ) {
-        return { userId, deltaMs };
-      }
-      return acc;
-    },
-    { userId: "", deltaMs: Number.MAX_SAFE_INTEGER },
-  );
+  // Sort buzzes by user ID so that we always pick the same winner in case
+  // of a tie, and do not inadvertently rely on 'Map' iteration order.
```

Comment:
Can you put this above L53 (the `sort` line so it's clear it refers to that function, not the overall chain of .sort, .reduce, etc.?

And how about a reword to:

```
Sort buzzes by user ID for deterministic results in case of a tie.
```